### ISSUE TITLE
[BUG] GetClips 

### ISSUE THREAD

**Issue by Jyojibh** [Sehelitar/Kick.bot#42]
So I've been messing around as I've been trying to do a ShoutOut command that plays the video Direct from the URL and i was hoping thats exactly what i would be able to do with this HOWEVER!

Upon looking at the clip[x].video It gives a playlist of the .m3u8 file rather than the specific video, So I did some digging to do the old way of downloading the clip to see if it still does the same thing and it seems Kick has changed how this works as it does indeed give a "Playlist.m3u8" file instead of the clip name .mp4 like it used to.

I've asked in the Community Dev area on their discord, But it seems like this has changed not sure when?

![image](https://github.com/Sehelitar/Kick.bot/assets/8933677/6c273fa2-a633-42dc-a9f4-4f1d6f32da2f)

**Comment by Sehelitar** [comment]
Yeah, it's something Kick changed, can't say exactly when, but it actually makes sense :
* Their current API is not supposed to be used by a third party, so they don't care about breaking-changes outside of their code.
* mp4 is inefficient vs. HLS/DASH streams, even for vods.
Since OBS' Media source can play HLS streams, it shouldn't be a problem though.
Do you have a use case where this doesn't work?
This is not something I can fix anyway, the only way to get a mp4 file is through trancoding. 🫤

**Comment by Sehelitar** [comment]
Ok, I actually found out a way to force Kick to do the transcoding part and give us a link to a mp4 file. 👀
Is it still necessary though.

**Comment by Jyojibh** [comment]
So basically the idea behind what i was after was to do the Better Shoutouts from Twitch, Basically if i can get that link to wherever the video is stored and the actual file link not to the small clip player itself, I can then have Streamerbot add the file link to the source and then it plays and then clears the source again before turning off.

Hopefully that makes sense 😛

**Comment by Jyojibh** [comment]
As it currently stands it just links to the small clip player but wont play... And then if you use the playlist file location its just broken and doesnt actually play at all..

**Comment by Sehelitar** [comment]
Alright then, I'll see what I can do. If the method I found is usable, I'll try to add it to the next release.

**Comment by Jyojibh** [comment]
The method works Perfectly for what it should do! I think its more just Kick messing with things! So now its kinda ruined what i had planned 😅 Because I wanted to be able to take the direct URL and have it run into a VLC / Media Player run the URL and then have it remove said URL until the next shoutout. But sadly because of the changes to that M3u8 file they had... It no longer works as i had intended it to.

**Comment by Jyojibh** [comment]
AMAZING! My guy you are truly gods work! Also was meaning to ask do you have a discord group at all ?

**Comment by Jyojibh** [comment]
Just an update as well, As i've got good standing with Kick Support on my Dashboard, After requesting Direct Link access like twitch does this was their response.

> We appreciate your understanding and cooperation!
>
>Your feedback and suggestions are greatly appreciated, and we would like to inform you that we have forwarded the direct links suggestion to our relevant team for further evaluation.
>
>While we cannot guarantee immediate implementation, please know that your voice matters to us, and we are glad that you reached out to us! 💚
